src/components/Demo/DemoCreate/index.tsx

```diff
@@ -59,3 +59,5 @@ export const DemoCreate: FunctionComponent = () => {
     </>
   );
 };
+
+export default DemoCreate;
```

Comment:
why ?

Reply:
issit to be able to load with dynamic ?

Reply:
yea i was lazy haha but theres a way to load named exports with dynamic too so i changed it already